src/tasks/onetime/DailyTask.py

```
@@ -81,16 +124,30 @@ def __init__(self, *args, **kwargs):
         )
         self.default_config.update({
+            "⭐地区建设": DailyRegionalRunner.DEFAULT_OPTIONS,
             "⭐传送到帝江号右侧传送点": True,
             "配置选择": "⭐⭐⭐ 默认",
             "发生异常时终止游戏": False,
             "仅退出游戏": False,
             "自动打开汇总文件": True,
         })
         self.config_description.update({
+            "⭐地区建设": (
+                "按地区执行所选操作：先据点兑换，再执行买卖货的买；启用买物资时，买完后切换到稳定物资需求购买，最后切回弹性需求物资执行卖。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)